mozilla_django_oidc/auth.py

```diff
@@ -99,7 +110,7 @@ def authenticate(self, code=None, state=None):
 
         # Validate the token
         token_response = response.json()
-        payload = self.verify_token(token_response.get('id_token'))
+        payload = self.verify_token(token_response.get('id_token'), **{'nonce': nonce})
```

Comment:
Completely agree. The only reason for going for keyword args, is because I didn't want to add the nonce keyword in the function signature. It's optional so if anybody wants to use it can pass as a keyword arg. Thoughts?